pkg/reconcilers/base_reconciler.go

```diff
@@ -0,0 +1,137 @@
+package reconcilers
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/3scale/3scale-operator/pkg/common"
+
+	"github.com/go-logr/logr"
+	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+)
+
+var log = logf.Log.WithName("reconcilers")
```

Comment:
Should this still exist?

Reply:
It is being used by public functions of the package. Those are not methods with `BaseController` type receiver

Reply:
:+1: it confused me seeing it in the base_reconciler.go file